code/modules/antagonists/fugitive/hunter.dm

```diff
@@ -1,6 +1,8 @@
 //The hunters!!
 /datum/antagonist/fugitive_hunter
 	name = "Fugitive Hunter"
+	antag_role_type = ROLE_KEY_FUGITIVE_HUNTER
+	antag_major_bancheck_type = null
```

Comment:
also should be bannable

Reply:
`antag_major_bancheck_type = null` means they're not bannable. This means they'll not be filtered by `MAJOR_BANCHECK_ANTAGONIST` meanwhile all ghost spawns are already banchcked by `MAJOR_BANCHECK_GHOSTSPAWN`